Makefile

```diff
@@ -0,0 +1,21 @@
+.PHONY: local-dev-clean
+local-dev-clean:
+	sudo ./scripts/clean.sh
```

Comment:
do we need `sudo` here?

Reply:
possibly not, let me try without it

Reply:
Yeah it doesn't need it, which makes sense. I'm assuming quay needs it since it's deleting sockets.

Reply:
I removed it